Branding update for release 20254 (#2848)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-139-update-branding-for-release-2024-8058.md

---
date: ""

> title: Update branding for 2024 release

Reviewer: ```suggestion
title: Updated branding colors and logo
```

product_area: Application enablement & solutions
change_type:

>   - value: change-VSkj2iV9m

Reviewer: ```suggestion
  - value: change-2c7RdTdXo4
```

>   - value: change-2c7RdTdXo4

Reviewer: ```suggestion
  - value: change-QHu1GdukP
```

>     label: Fix

Reviewer: ```suggestion
    label: Improvement
```

>     label: Improvement

Reviewer: ```suggestion
    label: Feature
```

component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62342
version: 1018.503.139
---

> We have updated the product's visual elements to reflect our company's new brand identity following the recent carve-out. These updates include removing previous company logos and implementing our new corporate color scheme across the application. These visual changes can be seen in various areas of the user interface, such as the login screen, header, and footer. This change is purely cosmetic and does not impact any functionality or existing user data.

Reviewer: I suggest to take the same text that we used in develop:

As part of ongoing efforts to modernize the look and feel of the Cumulocity UI, the branding colors and the logo have been updated and aligned with current design trends. For example, the brand logo has been relocated to the bottom of the navigator to enhance user focus on navigation elements, reduce visual competition with the app icon, and optimize space utilization. This change provides a refreshed visual appearance throughout the Cumulocity UI, without impacting existing functionality or user workflows, but it could imply a need to review any custom branding applied as some design tokens have been updated or removed. Additionally, the link color is no longer related to the branding color and needs to be updated if a branded link color is wanted.

> We have updated the product's visual elements to reflect our company's new brand identity following the recent carve-out. These updates include removing previous company logos and implementing our new corporate color scheme across the application. These visual changes can be seen in various areas of the user interface, such as the login screen, header, and footer. This change is purely cosmetic and does not impact any functionality or existing user data.

Reviewer: ```suggestion
As part of ongoing efforts to modernize the look and feel of the Cumulocity UI, the branding colors and the logo have been updated and aligned with current design trends. For example, the brand logo has been relocated to the bottom of the navigator to enhance user focus on navigation elements, reduce visual competition with the app icon, and optimize space utilization. This change provides a refreshed visual appearance throughout the Cumulocity UI, without impacting existing functionality or user workflows, but it could imply a need to review any custom branding applied as some design tokens have been updated or removed. Additionally, the link color is no longer related to the branding color and needs to be updated if a branded link color is wanted.
```

> As part of ongoing efforts to modernize the look and feel of the Cumulocity UI, the branding colors and the logo have been updated and aligned with current design trends. For example, the brand logo has been relocated to the bottom of the navigator to enhance user focus on navigation elements, reduce visual competition with the app icon, and optimize space utilization. This change provides a refreshed visual appearance throughout the Cumulocity UI, without impacting existing functionality or user workflows, but it could imply a need to review any custom branding applied as some design tokens have been updated or removed. Additionally, the link color is no longer related to the branding color and needs to be updated if a branded link color is wanted.

Reviewer: ```suggestion
As part of ongoing efforts to modernize the look and feel of the {{< product-c8y-iot >}} UI, the branding colors and the logo have been updated and aligned with current design trends. For example, the brand logo has been relocated to the bottom of the navigator to enhance user focus on navigation elements, reduce visual competition with the app icon, and optimize space utilization. This change provides a refreshed visual appearance throughout the {{< product-c8y-iot >}} UI, without impacting existing functionality or user workflows, but it could imply a need to review any custom branding applied as some design tokens have been updated or removed. Additionally, the link color is no longer related to the branding color and needs to be updated if a branded link color is wanted.
```
